dlls/rstrtmgr/tests/rstrtmgr.c

```
@@ -0,0 +1,139 @@
+
+    needed = 0;
+    count = 0;
+    ret = RmGetList(handle, &needed, &count, NULL, &reboot_reasons);
+    ok(ret == ERROR_MORE_DATA, "got %lu\n", ret);
+    ok(needed >= 1, "got needed %u\n", needed);
+    affected = calloc(needed, sizeof(*affected));
+    count = needed;
+    ret = RmGetList(handle, &needed, &count, affected, &reboot_reasons);
+    ok(ret == ERROR_SUCCESS, "got %lu\n", ret);
+    for (i = 0; i < count; ++i)
+        if (affected[i].Process.dwProcessId == process.dwProcessId) found = TRUE;
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)